cmd/podman/images/unmount.go

```diff
@@ -0,0 +1,71 @@
+package images
+
+import (
+	"fmt"
+
+	"github.com/containers/libpod/v2/cmd/podman/registry"
+	"github.com/containers/libpod/v2/cmd/podman/utils"
+	"github.com/containers/libpod/v2/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var (
+	description = `Image storage increments a mount counter each time an image is mounted.
+
+  When an image is unmounted, the mount counter is decremented. The image's root filesystem is physically unmounted only when the mount counter reaches zero indicating no other processes are using the mount.
+
+  An unmount can be forced with the --force flag.
+`
+	umountCommand = &cobra.Command{
+		Use:     "umount [flags] IMAGE [IMAGE...]",
+		Aliases: []string{"unmount"},
+		Short:   "Unmount an image's root filesystem",
+		Long:    description,
+		RunE:    unmount,
+		Example: `podman umount imgID
+  podman umount imgID1 imgID2 imgID3
+  podman umount --all`,
+	}
+)
+
+var (
+	unmountOpts entities.ImageUnmountOptions
+)
+
+func umountFlags(flags *pflag.FlagSet) {
+	flags.BoolVarP(&unmountOpts.All, "all", "a", false, "Umount all of the currently mounted images")
+	flags.BoolVarP(&unmountOpts.Force, "force", "f", false, "Force the complete umount all of the currently mounted images")
```

Comment:
`s/umount/unmount of/`

Reply:
Fixed in images/unmount.go and containers/unmount.go